CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 3.2.0 2021-11-10
```

Comment:
Could this title be changed to `## Unreleased` ? That way it can't get confused with the notes for identically-named 3.2.0 below. 

In other repositories, he infra team adds all changes under `Unreleased` and then changes that `Unreleased` line to `<release-version>` as part of the release procedure; perhaps that process could be useful here?

Reply:
I wasn't sure how to log this in the best way, but since there's some established way to do things like this, I will make changes